proto/errorpb.proto

```diff
@@ -12,47 +12,80 @@ option (rustproto.lite_runtime_all) = true;
 
 option java_package = "org.tikv.kvproto";
 
+// NotLeader is the error variant that tells a request be handle by raft leader 
+// is sent to raft follower or learner.
 message NotLeader {
+    // the current region ID
```

Comment:
What is current? The first letter of first word should be capitalized in a sentence.